model/database/common.yaml

```diff
@@ -21,3 +21,24 @@ groups:
           The `error.type` SHOULD match the error code returned by the database or the client library,
           the canonical name of exception that occurred, or another low-cardinality error identifier.
           Instrumentations SHOULD document the list of errors they report.
+
+  - id: attributes.db.cosmos.minimal
+    type: attribute_group
+    brief: 'Cosmos DB attributes'
+    attributes:
+      - ref: db.system
+        requirement_level: required
+      - ref: db.collection.name
+        requirement_level: required
+      - ref: db.namespace
+        requirement_level: required
+      - ref: db.operation.name
+        requirement_level: required
+      - ref: server.address
+        requirement_level: required
+      - ref: db.cosmosdb.status_code
+        requirement_level: required
+      - ref: db.cosmosdb.sub_status_code
+        requirement_level: required
+      - ref: db.cosmosdb.operation_type
```

Comment:
I don't think operation_type and name should be required - not always the need to have dimensions for both - and each dimension requires new time series - so, pretty heavy overhead

Reply:
If I want to view metrics specifically for read or write operations, the operation type dimension seems like the most appropriate choice. Since `operation_type` doesn't fluctuate much, it should not introduce any significant overhead—just an additional data point.

Reply:
In that case, lets remove the `operation_type` dimension. We can add it later if rquired.